File: docs/src/content/code/rust/convert_lcabyg.rs
use lcax_convert::lcabyg;

fn main() {
    let file_path = Path::new("../data/lcabyg_project.json");
    let contents = fs::read_to_string(file_path).unwrap();

    let project = lcabyg::parse::parse_lcabyg(&contents, None).unwrap();
}

File: docs/src/content/code/javascript/convertLCAbyg.ts
import {convertLCAbyg} from "lcax";
import fs from 'fs'

const lcabygData = fs.readFileSync('lcabyg_project.json')
const project = convertLCAbyg(lcabygData).project

// Output the LCAx formatted project to the console
console.log('LCAx', project)